### ISSUE TITLE
Typo in dimensions.json (JED)

### ISSUE THREAD

**Issue by ThiagoTGM** [FTBTeam/FTB-Interactions#818]
## Modpack Version: 
1.9.1

## Bug Report:
In the `dimensions.json` config for JustEnoughDimensions, there is a typo in the `worldinfo_onetime` section for the overworld dimension:

```json
"worldinfo_onetime": {
        "generatorName:": "customized",
        "generatorOptions": "..."
      }
```

namely `generatorName` is mistyped as `generatorName:`, leading to JED giving a "option not recognized" warning. Tbh I don't know how exactly this changes world generation since I'm not familiar with how generators work, but I thought it was a good idea to report it just in case.

## Known Fix: 
Remove the `:` from the key.

**Comment by Saereth** [comment]
yeah this isnt really impacting anything, will correct it for next update tho

**Comment by Saereth** [comment]
fixed in 62852435a84d061f1b92f375006352a64a320c9b
